opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -411,7 +413,7 @@ def _extract_attributes(
         )
     return ProtoSpan.Attributes(
         attribute_map=attributes_dict,
-        dropped_attributes_count=attributes_dict.dropped
+        dropped_attributes_count=attributes_dict.dropped  # type: ignore[attr-defined]
```

Comment:
As a follow up, should `dropped` be added to the `BoundedDict` type definition so that this pragma can be removed?

Reply:
Created an issue https://github.com/open-telemetry/opentelemetry-python/issues/1573